contracts/modules/LoanOpenings.sol

```diff
@@ -445,6 +437,42 @@ contract LoanOpenings is
         return (sentValues.newPrincipal, sentValues.collateralTokenSent); /// newPrincipal, newCollateral
     }
 
+    function _updateCollateralAfterTrade(
+        bytes32 loanId,
+        LoanParams memory loanParamsLocal,
+        MarginTradeStructHelpers.SentAddresses memory sentAddresses,
+        MarginTradeStructHelpers.SentAmounts memory sentValues,
+        bytes memory loanDataBytes
+    ) internal returns (MarginTradeStructHelpers.SentAmounts memory) {
+        // bytes32 loanIdLocal = loanId;
```

Comment:
can remove the commented code here?

Reply:
removed in this (commit)[https://github.com/DistributedCollective/Sovryn-smart-contracts/commit/30972ee37f140c050116180057818eeb97a6a356]